MAUI/CheckBox/Events.md

```diff
@@ -55,4 +55,38 @@ This event occurs when the value or state of the [`IsChecked`](https://help.sync
 {% endhighlight %}
 {% endtabs %}
 
-![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
\ No newline at end of file
+![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
+
+## StateChanging event
+
+The StateChanging event occurs when the state of [`IsChecked`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html#Syncfusion_Maui_Buttons_SfCheckBox_IsChecked) property is about to change in the Checkbox control. The StateChangingEventArgs provides the following properties:
```

Comment:
We trigger it, when we tap on the control alone. So, need to change content for this.

Reply:
I have changed the content for this.